dvc/commands/experiments/utils.py

```diff
@@ -0,0 +1,35 @@
+def add_rev_selection_flags(
+    experiments_subcmd_parser, command: str, default: bool = True
+):
+    experiments_subcmd_parser.add_argument(
+        "-A",
+        "--all-commits",
+        action="store_true",
+        default=False,
+        help=(
+            f"{command} all experiments in the repository "
+            "(overrides `--rev`)."
+        ),
+    )
+    msg = f"{command} experiments derived from the specified Git commit"
+    if default:
+        msg += "(instead of `HEAD`)."
+    experiments_subcmd_parser.add_argument(
+        "--rev",
+        type=str,
+        default=None,
+        help=msg,
+        metavar="<commit>",
+    )
+    experiments_subcmd_parser.add_argument(
+        "-n",
+        "--num",
+        type=int,
+        default=1,
+        dest="num",
+        metavar="<num>",
+        help=(
+            f"{command} list experiments from the `<rev>` commit and `num` "
+            "commits before it"
```

Comment:
Per https://github.com/iterative/dvc/pull/7245#discussion_r789400084
```suggestion
            f"{command} experiments from the `--rev` baseline and from `num` "
            "commits before it. Give `-1` to include all the parent commits."
```